sycl/test/host-interop-task/interop-task.cpp

```diff
@@ -0,0 +1,187 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out %threads_lib -lOpenCL
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// XFAIL: level0, cuda
```

Comment:
Could you add a comment why this test is expected to fail on Level Zero and CUDA, please?
```suggestion
// UNSUPPORTED: level0, cuda
```

Reply:
Done.